tools/c7n_kube/c7n_kube/actions/core.py

```diff
@@ -78,6 +78,13 @@ def patch_resources(self, client, resources, **patch_args):
 class PatchResource(PatchAction):
     """
     Patches a resource
+
+    Example for resource k8s.deployment:
+
+    - type: patch
```

Comment:
so the recommend here would be to make a fully valid policy for the example, we actually extract these doc strings and schema validate them during ci, to ensure that examples are valid.

the formatting syntax is slightly baroque, as its restructured text/sphinx style. as an example

```
    .. code-block:: yaml

        policies:
          - name: sqs-cross-account
            resource: sqs
            filters:
              - type: cross-account
```